lightning_pose/models/base.py

```diff
@@ -326,16 +314,27 @@ def configure_optimizers(self) -> dict:
                 gamma = MULTISTEPLR_GAMMA_DEFAULT
             else:
                 milestones = self.lr_scheduler_params.get(
-                    "milestones", MULTISTEPLR_MILESTONES_DEFAULT
-                )
+                    "milestones", MULTISTEPLR_MILESTONES_DEFAULT)
                 gamma = self.lr_scheduler_params.get("gamma", MULTISTEPLR_GAMMA_DEFAULT)
 
             scheduler = MultiStepLR(optimizer, milestones=milestones, gamma=gamma)
 
         else:
-            raise NotImplementedError(
-                "'%s' is an invalid LR scheduler" % self.lr_scheduler
-            )
+            raise NotImplementedError("'%s' is an invalid LR scheduler" % self.lr_scheduler)
+
+        return scheduler
+
+    def configure_optimizers(self) -> dict:
+        """Select optimizer, lr scheduler, and metric for monitoring."""
+
+        # get trainable params
+        params = self.get_parameters()
```

Comment:
I hope that this command works fine with the order of the parameters and the learning rate decays.

Reply:
I know that it does because I had a bug in this part of the code for a while - I accidentally named the inherited function `get_params` in the mhcrnn at first, and was getting terrible training results (because the crnn wasn't added to list of trainable params). once i fixed that model training worked beautifully.